getndazn/dazn-lambda-powertools
# fix(deps): aws-sdk should not be a direct dependency

## What did you implement:

Since aws-sdk is included in the lambda runtime, often users want to exclude it from their function
packages. When aws-sdk is imported as a direct dependency then users have to manually remove it from their node_modules directory.

## How did you implement it:

I've switched aws-sdk to a dev dep so it becomes the user's choice whether they want to fix the aws-sdk version and include it in their bundle or not.

## How can we verify it:

Install one of the packages, observe that aws-sdk is no longer in your node_modules dir.

## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
